Update active-directory-saas-tableauserver-tutorial.md

derekschauland opened the pull request:
I updated the idpattribute value to mailnickname - displayname doesn't make much sense in practical usage - a username being displayname doesn't make much sense as many times something similar to the samaccountname would fit better.  In testing this, and in production - user.mailnickname worked much better and was clearer. the displayed image should be updated to match.
![configure-sso-tableau](https://user-images.githubusercontent.com/19751585/36057560-42fa4df8-0dd5-11e8-9f59-3edfca67fcda.png)

azurecla commented:
Hi __@jastru__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

derekschauland committed:
Update active-directory-saas-tableauserver-tutorial.md

I updated the idpattribute value to mailnickname - displayname doesn't make much sense in practical usage - a username being displayname doesn't make much sense as many times something similar to the samaccountname would fit better.  In testing this, and in production - user.mailnickname worked much better and was clearer. the displayed image should be updated to match.
